src/Components/ArticleCard.jsx

```diff
@@ -1,4 +1,27 @@
-export default function ArticleCard({ article }) {
+import axios from "axios";
+import { useState } from "react";
+
+export default function ArticleCard({ article, setArticleList }) {
+
+  function handleLike(e) {
+    e.preventDefault();
+    setArticleList((currList) => {
+      const newList = currList.map((item) => {
+        if (item.article_id === +e.target.value) {
+          item.votes = item.votes + +e.target.id;
```

Comment:
Votes should not change while offline, ideally we want to display an error message when we are unable to patch 🙂